connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -174,8 +174,6 @@ protected void producerSendFailed(
             );
             commitTaskRecord(preTransformRecord, null);
         } else {
-            log.error("{} failed to send record to {}: ", WorkerSourceTask.this, topic, e);
-            log.trace("{} Failed record: {}", WorkerSourceTask.this, preTransformRecord);
             producerSendException.compareAndSet(null, e);
```

Comment:
Doesn't this mean we're potentially losing info about the topic which might be relevant to understanding the exception?

Reply:
The `TRACE`-level message is still [logged by the superclass](https://github.com/C0urante/kafka/blob/8ed7592f11f842fc2827a69008d1df3f34f36c78/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractWorkerSourceTask.java#L401-L407), and the `ERROR`-level message is still logged at `ERROR` level if the producer hasn't yet been closed. If the producer has been closed, the error is not worth logging at higher than `TRACE` since closing a producer causes all of its in-flight callbacks to complete with an error and the resulting noise to signal ratio is quite high..